templates/base/packages/nextjs/app/layout.tsx.template.mjs

```diff
@@ -1,15 +1,22 @@
-import { withDefaults } from "../../../../utils.js";
+import { deepMerge, stringify, withDefaults } from "../../../../utils.js";
+
+const defaultMetadata = {
+  title: "Scaffold-ETH 2 App",
+  description: "Built with 🏗 Scaffold-ETH 2"
+}
+
+const contents = ({ preConfigContent, metadataOverrides }) => {
```

Comment:
Should we call this `metadataOverrides` or call it `configOverrides` similar to other file like (Hardhat.conig or scaffold.config) to maintain similarity?

Reply:
I think this is good since in TEMPLATING.md

https://github.com/scaffold-eth/create-eth/blob/beta/contributors/TEMPLATING.md#rules-for-template-args

we mention to use `<name>Overrides`